R/utils-plot.R

```diff
@@ -374,3 +374,12 @@ formatMinMaxLabel <- function(ml, units){
   
   return(formatted_label)
 }
+
+#' Camel Case String
+#' 
+#' @description Formats the supplied string in CamelCasing
+#' @param string The string to format
+camelCase <- function(string) {
```

Comment:
only suggestion would be to make this `toCamelCase` since functions are performing actions. Suuuupppeerrr minor suggestion though.